docs/ref/applications.txt

```diff
@@ -219,6 +219,14 @@ Configurable attributes
 
     By default, this attribute isn't set.
 
+.. attribute:: AppConfig.default_auto_field
+
+    The implicit primary key type to add to models within this app. You can
+    use this to keep the historical :class:`~django.db.models.AutoField`
```

Comment:
This isn't really "historical". We're just providing a mechanism for people to have a different default auto field class.

Reply:
Wasn't the idea to migrate to using `BigAutoField` as the default in a future Django release? I've updated the wording though as it's currently not historical.